tests/storage/test_data_import_cron.py

```diff
@@ -205,6 +208,63 @@ def second_object_cleanup(
     resource_class(namespace=namespace.name, name=second_object_name).clean_up()
 
 
+@pytest.fixture()
+def data_import_cron_pvc_source(
+    unprivileged_client,
+    data_import_cron_namespace,
+    storage_class_with_filesystem_volume_mode,
+    pvc_for_source_data_import_cron,
+):
+    # cron_template_spec = golden_images_data_import_cron_spec.template.spec
+    with DataImportCron(
+        name="pvc-import-cron",
+        namespace=data_import_cron_namespace.name,
+        schedule="*/1 * * * *",
+        managed_data_source="pvc-import-datasource",
+        annotations=BIND_IMMEDIATE_ANNOTATION,
+        template={
+            "spec": {
+                "source": {
+                    "pvc": {
+                        "name": pvc_for_source_data_import_cron.name,
+                        "namespace": pvc_for_source_data_import_cron.namespace,
+                    }
+                },
+                "sourceFormat": "pvc",
+                "volumeMode": "Filesystem",
+                "storage": {
+                    "storageClassName": storage_class_with_filesystem_volume_mode,
+                    "accessModes": [PersistentVolumeClaim.AccessMode.RWO],
+                    "resources": {"requests": {"storage": "10Gi"}},
+                },
+            }
+        },
+    ) as data_import_cron:
+        wait_for_data_import_cron_ready_and_updated(
+            namespace=data_import_cron_namespace.name, name=data_import_cron.name
+        )
+        yield data_import_cron
+
+
+@pytest.fixture()
+def pvc_for_source_data_import_cron(
+    namespace, storage_class_with_filesystem_volume_mode, storage_class_name_scope_module
+):
+    with PersistentVolumeClaim(
+        name="source-pvc-test",
+        namespace=namespace.name,
+        accessmodes=PersistentVolumeClaim.AccessMode.RWO,
+        size="1Gi",
+        storage_class=storage_class_with_filesystem_volume_mode,
+    ) as pvc:
```

Comment:
This PVC is empty, right? We know that the use case is creating a VM, setting it up, and then using its PVC as a source for the DataImportCron. 

The best way to get a PVC that can be used as a boot source for a VM - is to create a DV that will import the OS image into a PVC.